imos_bgc_db/NRS_Phyto_Raw.schema.yaml

```diff
@@ -0,0 +1,27 @@
+fields:
+  - name: SAMPLE
+    description: Sample id (references NRS_Samp table)
+    type: integer
```

Comment:
The minimum we need to specify for each field is `name` and `type` (though type defaults to string if not specified).
`description` is just for internal documentation. It is not used for validation, so not essential.